src/app.module.ts

```diff
@@ -1,9 +1,25 @@
 import { Module } from '@nestjs/common';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import { StudentModule } from './student/student.module';
+import { TypeOrmModule } from "@nestjs/typeorm";
+import { config } from "./config/config";
 
 @Module({
-  imports: [],
+  imports: [
+    TypeOrmModule.forRoot({
```

Comment:
To samo robił Michał w tasku #7. Trzeba zaczekać na jego zmiany, zrobić  pull i działać na kodzie z jego zmianami.

Reply:
Obaj tego potrzebujemy do swoich tasków, więc IMO ten kto merguje poźniej po prostu rozwiązuje konflikty.